crates/as-if-std/Cargo.toml

```diff
@@ -15,15 +15,12 @@ bench = false
 cfg-if = "1.0"
 rustc-demangle = "0.1.21"
 libc = { version = "0.2.146", default-features = false }
-addr2line = { version = "0.20.0", default-features = false, optional = true }
-miniz_oxide = { version = "0.7", default-features = false }
 
-[dependencies.object]
-version = "0.31.1"
-default-features = false
-optional = true
-features = ['read_core', 'elf', 'macho', 'pe', 'unaligned', 'archive']
+[target.'cfg(not(all(windows, target_env = "msvc", not(target_vendor = "uwp"))))'.dependencies]
+miniz_oxide = { version = "0.7.0", optional = true, default-features = false }
+addr2line = { version = "0.20.0", optional = true, default-features = false }
+object = { version = "0.31.1", default-features = false, optional = true, features = ['read_core', 'elf', 'macho', 'pe', 'unaligned', 'archive'] }
```

Comment:
I would prefer we keep this as an "exploded-out" table here, as recent events have demonstrated that: it is extremely likely this will be diffed and that it may only diff a single field, so I would prefer to minimize the number of merge conflicts we have. I realize this means it looks like

```toml
[target.'cfg(not(all(windows, target_env = "msvc", not(target_vendor = "uwp"))))'.dependencies.object]
version = "0.32.0"
default-features = false
optional = true
features = ['read_core', 'elf', 'macho', 'pe', 'unaligned', 'archive']
```

Reply:
For some reason, `optional = true` exist there, but not in top Cargo.toml